### ISSUE TITLE
Drippy Crashing 1.19.3 Client

### ISSUE THREAD

**Issue by Zelioth** [Keksuccino/Drippy-Loading-Screen#34]
**Describe the bug**
Having your mods enabled causes the client to crash on launch

**To Reproduce**
Steps to reproduce the behavior (e.g.):
Install mods
run client

**Expected behavior**
To work correctly

**Game Log**
https://gist.github.com/Zelioth/97321762a042327470d8a2c69fceb9b6

**Screenshots**
If screenshots would help to explain the issue, please include these.

**Basic Informations (please complete the following information):**
Windows
Minecraft 1.19.3
Drippy 2.1.0 
Fancy 2.13.3
Kon 1.6.0

**Comment by Keksuccino** [comment]
Please try to load the game with only Drippy + dependencies installed. Could be a mod conflict.

**Comment by Keksuccino** [comment]
~ Just a note, please ignore ~

```
java.lang.IllegalStateException: Rendersystem called from wrong thread
	at TRANSFORMER/minecraft@1.19.3/com.mojang.blaze3d.systems.RenderSystem.m_187556_(RenderSystem.java:174)
	at TRANSFORMER/minecraft@1.19.3/com.mojang.blaze3d.systems.RenderSystem.m_187555_(RenderSystem.java:158)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.renderer.texture.AbstractTexture.m_117963_(AbstractTexture.java:54)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.font.FontTexture.<init>(FontTexture.java:28)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.font.FontSet.m_232556_(FontSet.java:163)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.font.glyphs.SpecialGlyphs.m_213604_(SpecialGlyphs.java:53)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.font.FontSet.m_95071_(FontSet.java:53)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.font.FontManager$1.m_95041_(FontManager.java:134)
	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.font.FontManager$1.m_5787_(FontManager.java:132)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.font.FontManager$1.m_5787_(FontManager.java:48)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.server.packs.resources.SimplePreparableReloadListener.invokeApplyDrippy(SimplePreparableReloadListener.java)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.screens.LoadingOverlay.loadFonts(LoadingOverlay.java:742)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.screens.LoadingOverlay.handler$zno000$onConstruct(LoadingOverlay.java:574)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.gui.screens.LoadingOverlay.<init>(LoadingOverlay.java:58)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.Minecraft.m_168019_(Minecraft.java:874)
	at TRANSFORMER/minecraft@1.19.3/net.minecraft.client.Minecraft.m_91391_(Minecraft.java:856)
	at TRANSFORMER/modelfix@1.10/net.mehvahdjukaar.modelfix.ModelFix.lambda$init$1(ModelFix.java:61)
	at TRANSFORMER/modelfix@1.10/net.mehvahdjukaar.modelfix.moonlight_configs.ConfigSpec.onRefresh(ConfigSpec.java:62)
	at TRANSFORMER/modelfix@1.10/net.mehvahdjukaar.modelfix.moonlight_configs.forge.ConfigSpecWrapper.onConfigChange(ConfigSpecWrapper.java:147)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:260)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:252)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:315)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:296)
	at LAYER PLUGIN/javafmllanguage@1.19.3-44.1.8/net.minecraftforge.fml.javafmlmod.FMLModContainer.lambda$new$0(FMLModContainer.java:42)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.ModContainer.lambda$dispatchConfigEvent$13(ModContainer.java:150)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.ModContainer.dispatchConfigEvent(ModContainer.java:150)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.config.ModConfig.fireEvent(ModConfig.java:77)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.config.ConfigTracker.openConfig(ConfigTracker.java:62)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.config.ConfigTracker.lambda$loadConfigs$1(ConfigTracker.java:50)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at java.base/java.util.Collections$SynchronizedCollection.forEach(Collections.java:2131)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.config.ConfigTracker.loadConfigs(ConfigTracker.java:50)
	at TRANSFORMER/forge@44.1.8/net.minecraftforge.fml.core.ModStateProvider.lambda$new$1(ModStateProvider.java:67)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.DistExecutor.unsafeRunWhenOn(DistExecutor.java:111)
	at TRANSFORMER/forge@44.1.8/net.minecraftforge.fml.core.ModStateProvider.lambda$new$3(ModStateProvider.java:66)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.ModLoader.lambda$dispatchAndHandleError$23(ModLoader.java:201)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.ModLoader.dispatchAndHandleError(ModLoader.java:201)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.ModLoader.lambda$loadMods$17(ModLoader.java:185)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at LAYER PLUGIN/fmlcore@1.19.3-44.1.8/net.minecraftforge.fml.ModLoader.loadMods(ModLoader.java:185)
	at TRANSFORMER/forge@44.1.8/net.minecraftforge.client.loading.ClientModLoader.lambda$startModLoading$5(ClientModLoader.java:120)
	at TRANSFORMER/forge@44.1.8/net.minecraftforge.client.loading.ClientModLoader.lambda$createRunnableWithCatch$4(ClientModLoader.java:111)
	at TRANSFORMER/forge@44.1.8/net.minecraftforge.client.loading.ClientModLoader.startModLoading(ClientModLoader.java:120)
	at TRANSFORMER/forge@44.1.8/net.minecraftforge.client.loading.ClientModLoader.lambda$onResourceReload$2(ClientModLoader.java:102)
	at TRANSFORMER/forge@44.1.8/net.minecraftforge.client.loading.ClientModLoader.lambda$createRunnableWithCatch$4(ClientModLoader.java:111)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

**Comment by Keksuccino** [comment]
@Zelioth Please try if it works without that mod installed: `net.mehvahdjukaar.modelfix.ModelFix`

**Comment by Keksuccino** [comment]
Fixed in Drippy v2.1.1!
